audio/src/main/java/edu/artic/audio/AudioNavViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package edu.artic.audio
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.annotation.UiThread
+import android.support.v4.app.FragmentManager
+import androidx.navigation.fragment.NavHostFragment
+import edu.artic.media.ui.NarrowAudioPlayerFragment
+import edu.artic.viewmodel.BaseViewModel
+import javax.inject.Inject
+
+
+/**
+ * Representation of excess [AudioActivity] logic.
+ */
+class AudioNavViewModel @Inject constructor() : BaseViewModel() {
```

Comment:
in this case the viewmodel is not really doing anything useful for testability. We should be exposing a `Subject` for clicks on the adapter, that then perform the business logic that populates another `Subject` that contains the text, that reports back to the activity.